package.json

```diff
@@ -39,7 +40,8 @@
     "jquery": "2.2.1",
     "lodash": "4.15.0",
     "n3": "0.6.1",
-    "webcola": "3.1.3"
+    "webcola": "3.1.3",
+    "whatwg-fetch": "^2.0.2"
```

Comment:
Ok, I think it's actually a good thing to move away from JQuery. I think we should specify a fixed version though (without `^`).